client/js/components/procedure/DpSimplifiedNewStatementForm.vue

```diff
@@ -42,7 +42,7 @@
             <dp-radio
               name="r_role"
               value="0"
-              data-cy="simplifiedNewStatementForm:citizenButton"
+              data-cy="roleInput-role.citizen"
```

Comment:
I thought we use `:` instead of `.` or `-` ?!?

Reply:
We maked these changes to be similar to other projects.
See:
- DpAutofillSubmitterData.vue:241
- DpAutofillSubmitterData.vue:245

Reply:
then we should change the other way round, so we get closer to our conventions, not more far away.

Reply:
https://github.com/demos-europe/demosplan-core/pull/3280/commits/043cb08ee5af62384ef51fcdce528dd1687578e9

Reply:
I fine this way is better for all projects:
```
data-cy="roleInput:privatperson"
data-cy="roleInput:institution"
```

Reply:
See https://github.com/demos-europe/demosplan-core/pull/3280/commits/55c7c8ce87c7416b569ab65c762cfa4e237eddcd